tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -105,7 +105,8 @@ def test_tqdm_progress_bar_totals(tmpdir):
 
     # check that the test progress bar is off
     assert 0 == bar.total_test_batches
-    assert bar.test_progress_bar is None
+    with pytest.raises(TypeError, match="test_progress_bar` .* not been set"):
+        assert bar.test_progress_bar is None
```

Comment:
This is technically a breaking change. Avoiding this would mean adding the `None` checks to all hooks inside `TQDMProgressBar`